src/simpleGit.ts

```diff
@@ -90,7 +90,36 @@ export class SimpleGit extends GitManager {
     async commitAll(message: string): Promise<number> {
         if (this.plugin.settings.updateSubmodules) {
             this.plugin.setState(PluginState.commit);
-            await this.git.subModule(["foreach", "--recursive", `git add -A && if [ ! -z "$(git status --porcelain)" ]; then git commit -m "${await this.formatCommitMessage(message)}"; fi`], (err) => this.onError(err));
+            this.git.outputHandler(async (x, y, z) => {
+                let body = ""
+                let root = this.app.vault.adapter.basePath
```

Comment:
`root` is since #209 not just the basepath

Reply:
Oh, my bad, ill fix that now